server/chat-plugins/datasearch.js

```diff
@@ -1212,12 +1255,29 @@ function runMovesearch(target, cmd, canAll, message) {
 	}
 	if (results.length > 1) {
 		results.sort();
+		if (sort) {
+			let prop = sort.slice(0, -1);
+			let direction = sort.slice(-1);
+			results.sort((a, b) => {
+				let move1prop = dex[toId(direction === '+' ? a : b)][prop], move2prop = dex[toId(direction === '+' ? b : a)][prop];
```

Comment:
```suggestion
				let move1prop = dex[toId(a)][prop];
				let move2prop = dex[toId(b)][prop];
```

I thought we had a linter rule for this. Maybe it's code only.

Also, don't check `direction` here.